BF luis:convert error while deploying Calendar Skill it-it and de-de

I still get these errors when trying to deploy Calendar Skill for IT and DE:

> Converting it-it Calendar LU file ...[ERROR] line 1858:0 - line 1858:20: PREBUILT entity 'personName' is not available for the requested locale 'it-it'
> Converting de-de Calendar LU file ...[ERROR] line 1880:0 - line 1880:20: PREBUILT entity 'personName' is not available for the requested locale 'de-de'

Using:
- latest version of the botframework-skills code downloaded on 3/7/2020
- botframework-cli version 4.9.1

Ciao,
Alessandro.

_Originally posted by @GitDruid in https://github.com/microsoft/botframework-cli/issues/412#issuecomment-654043362_

--- a/skills/csharp/calendarskill/Deployment/Resources/LU/de-de/Calendar.lu
+++ b/skills/csharp/calendarskill/Deployment/Resources/LU/de-de/Calendar.lu
@@ -1877,8 +1877,6 @@ $FloorNumber:simple
 
 $PREBUILT:datetimeV2
 
-$PREBUILT:personName
-
 $PREBUILT:ordinal
 
 

--- a/skills/csharp/calendarskill/Deployment/Resources/LU/es-es/Calendar.lu
+++ b/skills/csharp/calendarskill/Deployment/Resources/LU/es-es/Calendar.lu
@@ -1856,8 +1856,6 @@ $FloorNumber:simple
 
 $PREBUILT:datetimeV2
 
-$PREBUILT:personName
-
 $PREBUILT:ordinal
 
 

--- a/skills/csharp/calendarskill/Deployment/Resources/LU/fr-fr/Calendar.lu
+++ b/skills/csharp/calendarskill/Deployment/Resources/LU/fr-fr/Calendar.lu
@@ -1864,8 +1864,6 @@ $FloorNumber:simple
 
 $PREBUILT:datetimeV2
 
-$PREBUILT:personName
-
 $PREBUILT:ordinal
 
 

--- a/skills/csharp/calendarskill/Deployment/Resources/LU/it-it/Calendar.lu
+++ b/skills/csharp/calendarskill/Deployment/Resources/LU/it-it/Calendar.lu
@@ -1855,8 +1855,6 @@ $FloorNumber:simple
 
 $PREBUILT:datetimeV2
 
-$PREBUILT:personName
-
 $PREBUILT:ordinal
 
 

